docs/pages/kubernetes-access/getting-started/cluster.mdx

```diff
@@ -5,12 +5,18 @@ description: Getting started with Teleport. Let's deploy Teleport in a Kubernete
 
 # Getting Started
 
-Let's deploy Teleport in a Kubernetes with SSO and Audit logs:
+Let's deploy Teleport in a Kubernetes cluster with SSO and Audit logs:
```

Comment:
Readers might get a little bit confused that we're using the same kubernetes cluster as both:

- a place to deploy teleport
- the resource that teleport will provide access to

Can we try to clarify this a bit? Maybe saying something like:

"Teleport can provide secure, unified access to your Kubernetes clusters. This guide assumes you don't have an existing Teleport cluster running, and will deploy Teleport on the same Kubernetes cluster you'll provide access to."